lib/ansible/modules/system/systemd.py

```diff
@@ -415,7 +415,7 @@ def main():
             # check systemctl result or if it is a init script
             if rc == 0:
                 enabled = True
-            elif rc == 1:
+            elif rc > 1:
```

Comment:
found and is_sysv variables allow us to handle the rc better, `rc == 1 and found and not is_sysv ...  else`

Reply:
might still need to check stdout/err for errors